CODEOWNERS

```diff
@@ -0,0 +1,73 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*       @lgirdwood @plbossart @mmaka1 @lbetlej
+
+# Order is important. The last matching pattern has the most precedence.
+# So if a pull request only touches javascript files, only these owners
+# will be requested to review.
+
+# include files
+include/sound/sof/control.h			@ranj063 @singalsu @dbaluta
+include/sound/sof/dai.h				@ranj063 @singalsu @juimonen @dbaluta
+include/sound/sof/dai-intel.h			@ranj063 @singalsu @juimonen
+include/sound/sof/header.h			@lyakh @ranj063 @keyonjie @juimonen
+include/sound/sof/info.h			@lyakh @ranj063 @keyonjie @juimonen
+include/sound/sof/pm.h				@lyakh @ranj063 @keyonjie @juimonen
+include/sound/sof/stream.h			@lyakh @ranj063 @keyonjie @juimonen
+include/sound/sof/topology.h			@ranj063 @singalsu @bardliao @juimonen
+include/sound/sof/trace.h			@juimonen @lyakh @kv2019i
+include/sound/sof/xtensa.h			@liakh @xiulipan
```

Comment:
@plbossart is this a typo here? @liakh -> @lyakh?

Reply:
yes, thanks